loadtest/tests.go

```diff
@@ -290,6 +291,135 @@ func createPost(c *EntityConfig, team *UserTeamImportData, channelId string) {
 	}
 }
 
+func createChannel(c *EntityConfig, teamId string, userId string) (*model.Channel, error) {
+	const (
+		PUBLIC_CHANNEL = iota + 1
+		PRIVATE_CHANNEL
+		DIRECT_CHANNEL
+		GROUP_CHANNEL
+	)
+
+	choices := []randutil.Choice{
+		{
+			Weight: int(c.LoadTestConfig.UserEntitiesConfiguration.PublicChannelCreationChance * 1000),
+			Item:   PUBLIC_CHANNEL,
+		},
+		{
+			Weight: int(c.LoadTestConfig.UserEntitiesConfiguration.PrivateChannelCreationChance * 1000),
+			Item:   PRIVATE_CHANNEL,
+		},
+		{
+			Weight: int(c.LoadTestConfig.UserEntitiesConfiguration.DirectChannelCreationChance * 1000),
+			Item:   DIRECT_CHANNEL,
+		},
+		{
+			Weight: int(c.LoadTestConfig.UserEntitiesConfiguration.GroupChannelCreationChance * 1000),
+			Item:   GROUP_CHANNEL,
+		},
+	}
+
+	choice, err := randutil.WeightedChoice(c.r, choices)
+	if err != nil {
+		return nil, err
+	}
+
+	if choice.Item == PUBLIC_CHANNEL || choice.Item == PRIVATE_CHANNEL {
+		channel := &model.Channel{}
+		channel.TeamId = teamId
+		channel.Name = makeChannelName(c.r.Int())
+		channel.DisplayName = makeChannelDisplayName(c.r.Int())
+
+		if choice.Item == PUBLIC_CHANNEL {
+			channel.Type = "O"
```

Comment:
Can we use the `model.*` constants for these instead?

Reply:
Yep